### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by wayungi**
Add mobile version
Add Desktop version

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    card: 'card',
+    backface: 'olara@gmail.com',
+    back: 'back',
+    imageDetails: ['images/olara.jpg', 'yori bonger\'s image', 'speaker-image'],
+    speakerBio: 'speaker-bio',
+    h3Class: 'speaker-name',
+    SpeakerName: 'Olara Peter',
+    pOneClass: 'profession',
+    pOneContent: 'Study masters in information technology at MakerereUniversity',
+    pTwoClass: 'role',
```
Comment (micheaol) [inline script.js:25]: Your app is not performing as expected. All pages, links, and images are not working as expected. The `speaker section` is not loading as expected. I got the error below after the app was loaded.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+      </div>  
+      <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+      <ul class="nav-list flex-row">
+        <li class="nav-item "><a href="#" class="active">About</a></li>
+        <li class="nav-item"><a href="#">program</a></li>
+        <li class="nav-item"><a href="#">Join</a></li>
+        <li class="nav-item"><a href="#">Sponsor</a></li>
```
Comment (micheaol) [inline index.html]: It was not possible to navigate between the pages. I couldn't navigate from the `Home` page to the `About` page. Also, your `mobile hamburger menu` is not responding to click so I am unable to check what the menu likes. Kindly fix it.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+      <div class="hamburger small-screen">
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+        <span class="hamburger-bar"></span>
+      </div>  
+      <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+      <ul class="nav-list flex-row">
+        <li class="nav-item "><a href="#" class="active">About</a></li>
+        <li class="nav-item"><a href="#">program</a></li>
+        <li class="nav-item"><a href="#">Join</a></li>
```
Comment (micheaol) [inline index.html]: Please ensure that your `navbar` is like the original design.  Please ensure to add the top short bar section with social media links, logout, and language links aligned to the right.

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+        <section class="partner-logos">
+          <h3>Partner</h3>
+          <span class="guide-bar"></span>
+          <div class="partners">
+            <p class="mtn">MTN</p>
+            <p class="airtel">Airtel</p>
+            <p class="capital">Capital FM</p>
+            <p class="sanyu">Sanyu Radio</p>
+            <p class="silk">Silk Events</p>
+          </div>
```
Comment (micheaol) [inline index.html]: The error at your `feature speaker section` prevents your `footer` from loading. So, I am unable to review the `footer`, `speaker section`, and `about page`. Kindly fix the error. Also, I am unable to navigate to your `about page`.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+            </div>
+            <p class="description">
+              Debru was born in Burundi but grew up in Belgium, 
+              moving to Uganda in 2010.
+            </p>
+          </div>
+          <div class="lecture program-panel">
+            <div class="programe-icon">
+              <img src="images/program_icon_02.png" alt="">
+              <p class="icon-title">CC Exhibtion</p>
```
Comment (micheaol) [inline index.html:68]: Please ensure that the pages look almost identical to the original design. Please use the same `color, fonts` and also implement the `hover effect` on the cards as in the original design. Please ensure that the pages use the exact same colors, for text, backgrounds, borders, etc, from the original design.

* Basic color swatch: #272A31, #EC5242, #D3D3D3

Your Implementation | Design
:---:|:---:
image | image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    </nav>
+    <div class="home">
+      <header class="title">
+        <p class="tagline spacing">"MTN Nyegenyege welcomes all"</p>
+        <h2 class="page-title page-title-1 spacing">THE MTN NYEGENYEGE FESTIVAL 2022</h2>
+        <p class="introduction spacing">
+          Nyege Nyege is a collective in Kampala, Uganda that 
+          promotes outsider music, primarily electronic, by African artists. 
+          It was founded in 2013 by expats Arlen Dilsizian and Derek Debru. 
+          In addition to organizing ongoing parties and an artist residency
```
Comment (micheaol) [inline index.html]: Please ensure that there's at least 1, but no more than 1, `h1` tag in each page according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @wayungi , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+          the collective runs.
+        </p>
+        <div class="event-date spacing">2022.8.12 (MON) ~ 18(SUN)</div>
+        <p class="venue spacing">The event will be held in Jinja at the Nile</p>
+      </header>
+      <section class="main-program">
+        <h2 class="program-title">main program</h2>
```
Comment (Danieldotcomcoder) [inline index.html:52]: [ OPTIONAL ]: It would be better if you add some margin up for this section so that the header can take the whole page ✔😀

**Review by Danieldotcomcoder — APPROVED** [review]
Hello @wayungi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

comp-2
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**